pkg/services/ngalert/notifier/external_alertmanager.go

```diff
@@ -121,30 +123,62 @@ func (am *externalAlertmanager) ListSilences(ctx context.Context, filter []strin
 	return res.Payload, nil
 }
 
-func (am *externalAlertmanager) GetStatus() apimodels.GettableStatus {
```

Comment:
The git diff is weird here, the `GetStatus()` method was moved below the three alert-related methods.